content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.getActualMaximum()'
+Description: 'A Calendar class method that returns the maximum value the specified calendar field could have, based on the time value of the calendar'
```

Comment:
```suggestion
Description: 'Returns the maximum value the given calendar field can have, conditional on the time value of the calendar.'
```

Reply:
The original description and example all appear to be directly copied from https://www.geeksforgeeks.org/calendar-getactualmaximum-method-in-java-with-examples/

All submitted entries must be original content. Copying content from another site is plagiarism, and is not permitted in docs. Please revise this entry and example to replace all copied content.